Sources/Features/Attributes/DefaultAttributes.swift

```diff
@@ -232,7 +237,7 @@ struct MetricsInfo: AttributesSource {
 
     var immutable: [String: Any?] {
         return MetricsInfo.isMetricsEnabled ?
-            ["application.version": Backtrace.applicationVersion,
+            ["application.version": Bundle.main.releaseVersionNumber,
```

Comment:
This should be the version of the client app using our library, so I think this is right. We should verify this though.